health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/components/CustomPieChart.js

```
@@ -0,0 +1,315 @@
+  const [isPieClicked, setIsPieClicked] = useState(false);
+  const [pieSelected, setPieSelected] = useState(null);
+  const [showTooltip, setShowTooltip] = useState(false);
+  const [drillDownId, setdrillDownId] = useState(null);
+  const [activeIndex, setActiveIndex] = useState(null);
+  const { startDate, endDate, interval } = getInitialRange();
+  const { projectTypeId } = Digit.Hooks.useQueryParams();
+  const selectedProjectTypeId = projectTypeId ? projectTypeId : Digit.SessionStorage.get("selectedProjectTypeId");
+
+  const today = new Date();
+  today.setHours(0, 0, 0, 0);
+  let todayDate = today;
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'todayDate' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)